Lab_2/non_compressed_text.txt

```diff
@@ -1,6 +1,6 @@
 He was an old man who fished alone in a skiff in the Gulf Stream and he had gone
 eighty-four days now without taking a fish. In the first forty days a boy had been with him.
-But after forty days without a fish the boy’s parents had told him that the old man was
+But after forty days without a fish the boy's parents had told him that the old man was
```

Comment:
The **’** char is not in the ASCII table, let's switch it to the common **'** instead.